CHANGES.md

```diff
@@ -7,6 +7,7 @@
 
 New Features:
 
+* [#933](https://github.com/ckeditor/ckeditor-dev/issues/1441): [Magic Line](https://ckeditor.com/cke4/addon/magicline) line element can now be identifiable using `data-cke-magic-line="1"` attribute.
```

Comment:
Wrong issue number here. Please also remember that new entry should be added on the end of existing list.

Reply:
Another thing is that this fix is targeting **major** (next **major** is `4.9.0`) so the changelog entry should be put under **CKEditor 4.9** not **CKEditor 4.8.1**.